vintf/vendor.nxp.nxpese.xml

```diff
@@ -0,0 +1,11 @@
+<manifest version="1.0">
```

Comment:
You're missing  type ` "device"` or it will fail building:

`File "device/sony/common/vintf/vendor.nxp.nxpese.xml" is not a valid device manifest (but the first file is a valid device manifest). Error: Could not find/parse attr with name "type" and value "" for element <manifest>`